sdk/lib/opentelemetry/sdk/resources/resource.rb

```diff
@@ -15,19 +15,22 @@ class << self
 
           # Returns a newly created {Resource} with the specified attributes
           #
-          # @param [Hash{String => String, Numeric, Boolean} attributes Hash of key-value pairs to be used
+          # @param [Hash{String => String, Numeric, Boolean}] attributes Hash of key-value pairs to be used
           #   as attributes for this resource
+          # @param [String] Specifies the Schema URL that should be recorded in the emitted resource.
           # @raise [ArgumentError] If attribute keys and values are not strings
           # @return [Resource]
-          def create(attributes = {})
+          def create(attributes = {}, schema_url = '')
```

Comment:
I kind of wish we just used `nil` here instead of the empty string. The `schema_url` default for `InstrumentationScope` is `nil` and the difference in behaviour is troubling.

Reply:
I'll look into it!

Reply:
@fbogsany I changed that behavior; but I chose to interpret a blank string as if it were opaque in the merging behavior (that is `schema_url: nil` and `schema_url: ''` are not treated as equivalent in our merge, and we fall through to the unspecified behavior section of the spec). Let me know if that's a bad idea in your eyes.

Reply:
Also: I was going to make it a kwarg on the `Resource` class as well, but since we have `attributes = {}` we ran into failures like:

```
/Users/ahayworth/projects/opentelemetry-ruby/sdk/lib/opentelemetry/sdk/resources/resource.rb:24:in `create': unknown keyword: "service.name" (Argument
Error)
        from /Users/ahayworth/projects/opentelemetry-ruby/sdk/lib/opentelemetry/sdk/resources/resource.rb:38:in `default'
```

...which makes sense of course, it would be treating everything passed as an attribute as part of the kwargs.

I chose to just keep it a positional param there, but if there's a more elegant way to fix that up, we can do so.